hotness/consumers.py

```diff
@@ -284,8 +284,7 @@ def _handle_anitya_update(self, upstream, package, msg):
                                 else:
                                     self.new_triggered_task_ids[task_id] = [bz, None, str(upstream), str(package)]
                 else:
-                    note = 'Patching or scratch build for %s and version %s FAILED.\n' \
-                           'See for details' % (package, version)
+                    note = 'Patching or scratch build for %s and version %s FAILED.\n' % (package, version)
```

Comment:
True there.  So something instead like:

``Patching or scratch build for %s-%s failed." % (...``

I'll go ahead and commit that directly to the ``develop`` branch.